samples/bluetooth/peripheral_hids_mouse/src/main.c

```diff
@@ -554,6 +590,26 @@ static void auth_passkey_display(struct bt_conn *conn, unsigned int passkey)
 }
 
 
+static void auth_passkey_confirm(struct bt_conn *conn, unsigned int passkey)
+{
+	int err;
+
+	struct pairing_data_mitm pairing_data;
+
+	pairing_data.conn    = conn;
```

Comment:
Since you keep a ref to this conn object, please use `bt_conn_ref`, and when you clear the pointer use `bt_conn_unref`

Reply:
Ok, now this functions are used.

Reply:
Please go over, you missed some.

Reply:
Add missed ref to conn in central samples ;)